### PR TITLE
Support for DecompressionStream

### PR DESCRIPTION
**Pull request by keroami**
to either gunzip or inflate

### CODE REVIEW

File: src/decompression_stream_ffi.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { toBitArray } from "../gleam_stdlib/gleam.mjs"
+
+export const decompress = async (data, encoding) => {
+  const ds = new DecompressionStream(encoding);
+  const blob = new Blob([data.buffer])
+  const decompressedStream = blob.stream().pipeThrough(ds)
+
+  return await new Response(decompressedStream)
+        .blob()
+        .then(blob => blob.arrayBuffer())
```
Comment (CrowdHailer) [inline src/decompression_stream_ffi.mjs:10]: Can you not use await and then in the same context? I found it rather confusing.
My preference would be for all await style.

**Comment by CrowdHailer** [comment]
Thanks. Does it make sense to add the Compression stream at the same time?

File: src/decompression_stream_ffi.mjs (hunk 1 above)
Comment (keroami) [reply src/decompression_stream_ffi.mjs]: done. I'm getting more and more familiar with the JS way of writint this ;)

**Comment by keroami** [comment]
adding compression stream, now probably trivial :)
lemme do that tonight or tomorrow.

**Comment by keroami** [comment]
added compression stream.

took out the (simpli)file hello.txt.gz file and inlined a shorter version in the tests.

**Comment by CrowdHailer** [comment]
Thanks. I like the remove of simplifile as a dependency. publish in version `0.1.13`

**Pull request merged**